packages/wrangler/src/utils/ci-check.ts

```diff
@@ -0,0 +1,34 @@
+import ci from "ci-info";
```

Comment:
Created a `utils` directory, with the `ciCheck` in a similar way we handle `cfetch` which allows us to mock the local module in Jest setup rather than the dependency itself.

Reply:
this doesn't have to be in a new directory, we're trying to avoid "utils". keep it in src itself. cfetch has other files attached to it, hence the folder there.

Reply:
Done.